nepi/main/tests/factories.py

```diff
@@ -14,6 +14,7 @@ class Meta:
     password = factory.PostGenerationMethodCall('set_password', 'test')
     last_name = factory.Sequence(lambda n: "Last%d" % n)
     first_name = factory.Sequence(lambda n: "First%d" % n)
+    email = factory.Sequence(lambda n: "username%d@localhost" % n)
```

Comment:
I put a breakpoint in the [DenyFacultyView.send_denied_email](https://github.com/ccnmtl/nepi/blob/master/nepi/main/views.py#L591) and traced through send_mail until I hit this [condition](https://github.com/django/django/blob/1.11/django/core/mail/message.py#L338). I guessed at that point that the UserFactory was not creating the user with an email.

Looks like this commit changed the behavior in 1.11: https://github.com/django/django/commit/8858631498bca33feb2acde71f9cf6ca2884b1bd